EpsilonAuxBMS/Src/AuxBMS.c

```diff
@@ -0,0 +1,340 @@
+#include "cmsis_os.h"
```

Comment:
This file is getting to be pretty busy, consider breaking it up into different files (One for each Task).